integration_tests/serverless-forwarder.yml

```diff
@@ -87,3 +89,66 @@ functions:
   providedHello:
     handler: provided_handler.hello
     runtime: provided
+
+resources:
+  Resources:
+    stepFunction2LogGroup:
+      Type: AWS::Logs::LogGroup
+      Properties:
+        LogGroupName: /aws/vendedlogs/states/stepFunctionLoggingConfigDynamicArn-Logs-dev
+    stepFunction3LogGroup:
+      Type: AWS::Logs::LogGroup
+      Properties:
+        LogGroupName: /aws/vendedlogs/states/stepFunctionNoLoggingConfigStaticArn-Logs-dev
+
+stepFunctions:
+  stateMachines:
+    stepfunction1:
+      name: stepFunctionNoLoggingConfig
+      definition:
+        StartAt: State1
+        States:
+          State1:
+            Type: Task
+            Parameters:
+              Payload.$: "States.JsonMerge($$, $, false)"
```

Comment:
Since SF and Lambda span linking instrumentation is not implemented in this PR, maybe we can remove these `Payload.$: "States.JsonMerge($$, $, false)"` lines to make the PR simpler.

Reply:
I removed the payloads from the step function parameters in the integration test